armi/reactor/blueprints/reactorBlueprint.py

```diff
@@ -235,10 +234,11 @@ def _modifyGeometry(self, container, gridDesign):
         # all cases should have no edge assemblies. They are added ephemerally when needed
         from armi.reactor.converters import geometryConverters
 
-        runLog.header("=========== Applying Geometry Modifications ===========")
-        converter = geometryConverters.EdgeAssemblyChanger()
-        converter.scaleParamsRelatedToSymmetry(container)
-        converter.removeEdgeAssemblies(container)
+        if not container.isFullCore:
+            runLog.header("=========== Applying Geometry Modifications ===========")
```

Comment:
My recommendation would be to leave this header statement outside of the conditional. That way if more geometry conversions/scaling/etc. are done in the future from within other conditionals, then the header doesn't need to be repeated.